hyrax.gemspec

```
@@ -45,11 +45,10 @@ SUMMARY
   spec.add_dependency 'draper', '~> 4.0'
   spec.add_dependency 'dry-logic', '~> 1.5'
   spec.add_dependency 'dry-container', '~> 0.11'
-  spec.add_dependency 'dry-events', '~> 1.0.0'
-  spec.add_dependency 'dry-equalizer', '~> 0.2'
-  spec.add_dependency 'dry-monads', '~> 1.5'
-  spec.add_dependency 'dry-struct', '~> 1.0'
-  spec.add_dependency 'dry-validation', '~> 1.3'
+  spec.add_dependency 'dry-events', '~> 1.0.1'
+  spec.add_dependency 'dry-monads', '~> 1.6.0'
+  spec.add_dependency 'dry-struct', '~> 1.6.0'
+  spec.add_dependency 'dry-validation', '~> 1.10.0'
```

We can allow minor upgrades, but keep the events upgrade.
```suggestion
  spec.add_dependency 'dry-events', '~> 1.0', '>= 1.0.1'
  spec.add_dependency 'dry-monads', '~> 1.6'
  spec.add_dependency 'dry-struct', '~> 1.6'
  spec.add_dependency 'dry-validation', '~> 1.10'
```